packages/manager/modules/nutanix/src/dashboard/constants.js

```diff
@@ -11,10 +11,16 @@ export const GUIDE_URL = {
   },
 };
 
-export const NEW_CLUSTER_PLAN_CODE = [
-  'nutanix-standard',
-  'nutanix-advanced',
-  'nutanix-byol',
+export const OLD_CLUSTER_PLAN_CODE = [
+  'nutanix-21hci01-byol',
+  'nutanix-21hci02-byol',
+  'nutanix-21hci03-byol',
+  'nutanix-pro-21hci01',
+  'nutanix-pro-21hci02',
+  'nutanix-pro-21hci03',
+  'nutanix-ultimate-21hci01',
+  'nutanix-ultimate-21hci02',
+  'nutanix-ultimate-21hci03',
```

Comment:
**question:** I dont like the idea of maintaining this info on Manager. Can't API provide whether the Nutanix is an old or new one?